sql/create.sql

```diff
@@ -0,0 +1,49 @@
+CREATE TABLE USERS
+(
+    uid         INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+    name        VARCHAR(64) NOT NULL,
+    gh_id       VARCHAR(16) NOT NULL,
+    gh_token    VARCHAR(256) NOT NULL, -- Encrypted GitHub access token

```

Comment:
We may have users without GH tokens.

Reply:
Since we'll be using OAuth to authenticate the users via GitHub, we will need the user's GitHub access token to verify its identity via the [`/user` endpoint](https://docs.github.com/en/rest/reference/users#get-the-authenticated-user). Since the token doesn't expire, we feel it's useful to store it during this procedure, as it can also be used to list the user's organizations, for example.